lms/product/family.py

```diff
@@ -16,3 +16,21 @@ class Family(str, Enum):
     @classmethod
     def _missing_(cls, _value):
         return cls.UNKNOWN
+
+    @classmethod
+    def from_launch(cls, lti_params: dict):
```

Comment:
Moved the method here from product.factory.

Now the method is not about "guessing" the product with different methods. It only looks at information available during launch.